maven/bnd-maven-plugin/README.md

```diff
@@ -77,17 +77,32 @@ although it will be valid, the bundle may not be *useful*.
 For further usage information, see the integration test projects under the included
 `src/test/resources/integration-test/test` directory.
 
-## Configuration Properties
+## Configuration Parameters
```

Comment:
You need to amend your commits so that each commit message includes a `Signed-off-by` as required by CONTRIBUTING.

Reply:
I just did that